benchmark/v0_6_0.js

```diff
@@ -18,7 +18,7 @@ var v0_6_0 = function(text, list, useWordBase) {
       item._trie = trie;
     }
 
-    trieIndices = trie.getIndices(text);
+    trieIndices = trie.getIndexes(text);
```

Comment:
Just a remark here, but `indices` is a correct plural for `index` (`indexes` is also correct).